lib/Queue.rb

```diff
@@ -1,22 +1,39 @@
 class Queue
+attr_accessor :store
```

Comment:
ahhh okay, I think I understand why an attr_accessor is not a good idea but I'm still a bit fuzzy on why attr_reader Isn't a good idea. If I wanted to see the queue contents in literal form, instead of calling ```ruby q.store # :store in attr_reader``` I would just do something like ```ruby puts q ```?